### PR TITLE
send downstream nodes in domain list, fix identity packet bug

### PR DESCRIPTION
**Pull request by birarda**

### CODE REVIEW

File: assignment-client/src/audio/AudioMixer.cpp [hunk 1]
```diff
@@ -103,6 +103,11 @@ AudioMixer::AudioMixer(ReceivedMessage& message) :
     );
 
     connect(nodeList.data(), &NodeList::nodeKilled, this, &AudioMixer::handleNodeKilled);
+    connect(nodeList.data(), &NodeList::nodeAdded, this, [this](SharedNodePointer node) {
```
Comment (Atlante45) [inline assignment-client/src/audio/AudioMixer.cpp]: `const SharedNodePointer&`?

File: assignment-client/src/avatars/AvatarMixer.cpp [hunk 2]
```diff
@@ -63,6 +63,12 @@ AvatarMixer::AvatarMixer(ReceivedMessage& message) :
 
     auto nodeList = DependencyManager::get<NodeList>();
     connect(nodeList.data(), &NodeList::packetVersionMismatch, this, &AvatarMixer::handlePacketVersionMismatch);
+    connect(nodeList.data(), &NodeList::nodeAdded, this, [this](SharedNodePointer node) {
```
Comment (Atlante45) [inline assignment-client/src/avatars/AvatarMixer.cpp]: `const SharedNodePointer&`?

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/767/HighFidelity-Beta-PR10709-ea6a38148c08fb6fa7b82ab2f260d913e6e063e7.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/767/HighFidelity-Beta-PR10709-ea6a38148c08fb6fa7b82ab2f260d913e6e063e7.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/769/HighFidelity-Beta-PR10709-d7724b90afa7de507fe45f7bf77aad6f7e249fa8.exe

**Pull request merged**